packages/terra-responsive-element/src/ResponsiveElement.jsx

```diff
@@ -99,13 +101,11 @@ class ResponsiveElement extends React.Component {
   }
 
   render() {
-    const { defaultElement, tiny, small, medium, large, huge, responsiveTo, ...customProps } = this.props;
+    if (this.state.element) {
+      return this.props[this.state.element] || null;
+    }
 
-    return (
-      <div {...customProps} ref={this.setContainer}>
-        {this.props[this.state.element]}
-      </div>
-    );
+    return <div ref={this.setContainer} />;
```

Comment:
Is this case for the initial render?

Reply:
Yes, on the initial render this will load into the dom, bind the appropriate resize observers, determine the appropriate element to display based on the available width and then rerender. It is not needed after the initial render.